profile-widget/divepixmapcache.cpp

```diff
@@ -49,7 +49,7 @@ DivePixmaps::DivePixmaps(int dpr) : dpr(dpr)
 	// The transparen pixmap is a very obscure feature to enable tooltips without showing a pixmap.
 	// See code in diveeventitem.cpp. This should probably be replaced by a different mechanism.
 	QPixmap transparentPixmap(lrint(4 * dprf), lrint(20 * dprf));
-	transparentPixmap.fill(QColor::fromRgbF(1.0, 1.0, 1.0, 0.01));
+	transparentPixmap.fill(QColor::fromRgbF(1.0f, 1.0f, 1.0f, 0.01f));
```

Comment:
Could reuse the inline function (or macro) in `color.h`.

Reply:
true

Reply:
done